### DISCUSSION TITLE
Frage zur Geräte Selektion mit "unignore"

### DISCUSSION THREAD

**Discussion by Baxxy13** [SukramJ/aiohomematic#1376] [Q&A] [answered]
unignore wurde ja mal erweitert um die Datenpunkte selektiv (z.B. für ein spezielles Gerät) zu aktivieren.

Grundsätzlich hatte ich einfach nur `LEVEL` drin, um z.B. beim HmIP-BROLL die reale Behanghöhe vom Statuskanal (:3) als eigene Entität zu haben.
Jetzt habe ich testweise umgestellt auf: `LEVEL@@HmIP-BROLL` und musste feststellen das das so nicht funktioniert.
Die LEVEL-Entität von Kanal:3 ist dann nicht mehr verfügbar.

Müsste das so funktionieren oder lese/deute/mache ich da was falsch?

**Comment by SukramJ** [comment]
Nur ein @

**Reply by Baxxy13** [reply]
Leider geht das auch mit nur einem @ nicht.
`LEVEL@HmIP-BROLL`

**Reply by SukramJ** [reply]
Versuch mal die 3. Variante:
`LEVEL@HmIP-eTRV-2:1:VALUES`

**Reply by Baxxy13** [reply]
`LEVEL@HmIP-BROLL:3:VALUES` geht leider auch nicht.

**Reply by SukramJ** [reply]
Muss ich mir mal demnächst anschauen.

**Reply by Baxxy13** [reply]
Alles klar, danke. 
Hab's erstmal wieder auf `LEVEL` gestellt.

**Reply by SukramJ** [reply, marked as answer]
Teste mal bitte mit der 1.54.0b0.
Das sollte funktionieren: `LEVEL@HmIP-BROLL:3:VALUES`
Das funktionierte nie, bzw. war eine Fehler in der [Doku](https://github.com/danielperna84/hahomematic/blob/devel/docs/unignore.md): `LEVEL@HmIP-BROLL`

**Reply by Baxxy13** [reply]
`LEVEL@HmIP-BROLL:3:VALUES` funktioniert nun mit der Beta. 
Sehr gut und wie immer... Danke für das fixe fixen. 😉 

In der aktualisierten Doku ist noch ein @ zuviel...
`LEVEL@@HmIP-eTRV-2:1:VALUES`
